### ISSUE TITLE
Memory Leak + High CPU Load + High GPU Load

### ISSUE THREAD

**Issue by Orel-A** [SoftFever/OrcaSlicer#4619]
### Is there an existing issue for this problem?

- [X] I have searched the existing issues

### OrcaSlicer Version

2.0.0 Beta

### Operating System (OS)

Windows

### OS Version

Windows 11 Pro 23H2

### Additional system information

CPU: Ryzen 7600X
GPU: AMD Radeon RX 5700 XT
RAM: DDR5 32GB 6000MHZ CL30

### Printer

Creality Ender-3 V3 KE

### How to reproduce

Honestly, for now I don't know how to reproduce this, but I'm opening this issue as it happened to me more than once and maybe more people have this problem.
What I do remember is that I opened two instances of Orca at the same time...

### Actual results

Screenshots of Task Manager:
![Screenshot 2024-03-21 181911](https://github.com/SoftFever/OrcaSlicer/assets/7490366/b75a4ce3-eea4-41c9-bc4a-45955b9a85b8)
![Screenshot 2024-03-21 181858](https://github.com/SoftFever/OrcaSlicer/assets/7490366/6bb1c1eb-961e-4119-a1e8-758ec55e3bd4)
![Screenshot 2024-03-21 181835](https://github.com/SoftFever/OrcaSlicer/assets/7490366/7612bc85-2c34-47dd-b32b-bf0f6e452aaa)

### Expected results

-

### Project file & Debug log uploads

[debug_Thu_Mar_21_18_26_58_310884.log.zip](https://github.com/SoftFever/OrcaSlicer/files/14704416/debug_Thu_Mar_21_18_26_58_310884.log.zip)

### Checklist of files to include

- [X] Log file
- [x] Project file

### Anything else?

Even after closing all running Orca instances, the memory leak is still present. Meaning I still have around 14GB RAM usage in Windows.

**Comment by Embeddi** [comment]
I'm also seeing 100% **GPU** load with 2.0.0-rc1 on the "Preview" tab. That happens even when OrcaSlicer window is minimized.

When switching back to "Prepare" then GPU load goes down to <1% again.

**Comment by UserDmitry** [comment]
Same issue here. When in Preview mode and top right menu scrolled down, some CPU cores are at 100% load. When menu is up - no additional cpu usage. 
![cpu burn](https://github.com/SoftFever/OrcaSlicer/assets/63342754/6894bbce-ecc6-4f83-99f4-8dd3b796ea4a)
When menu is up...
![cpu not burn](https://github.com/SoftFever/OrcaSlicer/assets/63342754/92423f5c-7f12-43b5-8ecb-edec82c91c39)

**Comment by cochcoder** [comment]
> It's happening when switching to Color scheme: Speed, in preview:
> 
> ![image](https://private-user-images.githubusercontent.com/7490366/322281505-6a66cfcf-693b-4cdf-8ae6-c17774b1ef68.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM0MDc0MjcsIm5iZiI6MTcxMzQwNzEyNywicGF0aCI6Ii83NDkwMzY2LzMyMjI4MTUwNS02YTY2Y2ZjZi02OTNiLTRjZGYtOGFlNi1jMTc3NzRiMWVmNjgucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDQxOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA0MThUMDIyNTI3WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9Yjg2M2JkZWZkNzI5YmIwYzdmOWFhNzZjMGVmMjE1NTMxNDUwNDg0MmMyOTljNDgyNTExMWM0YmY3ZWU0NzhiMCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.t0EGzuQWKXaTY1DBjLFH8-5Z_QgPeqmyZQSz1CGwnMA)
> 
> @SoftFever @cochcoder can you confirm?
> 
> Attaching an example: [Voronoi Pebble Coaster.zip](https://github.com/SoftFever/OrcaSlicer/files/14971123/Voronoi.Pebble.Coaster.zip)

I do notice that more of my CPU cores would be used at higher percentages as well when switching to color, I have also noticed that the interface gets very laggy when view large objects even though the CPU usage is at about 5-10% and Prusa Slicer doesn't have this issue on my system.

**Comment by dstulken** [comment]
I can also confirm high CPU when switching to "Speed" in the preview.
MacBook Air M1, Orca Slicer 2.0.0

On the particular model on my plate, CPU usage with the "Line Type" view is using 19-25% CPU.   When I switch to "Speed", it jumps to 110% CPU.   The "Filament" view also is way up there (92% CPU), but others seem to be <25%.

**Comment by mattyg2301** [comment]
Same here

**Comment by CCS86** [comment]
@SoftFever can you reproduce?

**Comment by Imploo** [comment]
Same here, preview tab slows laptop down even in other programs. 100% GPU usage, graph "Copy". I have a dedicated Nvidia GPU.

**Comment by celulari** [comment]
Still happens on Orca 2.1.1

![image](https://github.com/user-attachments/assets/e6e41576-c14c-4d76-8487-d3af73e6f896)
![image](https://github.com/user-attachments/assets/4248e780-8e28-4c5a-b7d4-e6c4c96244da)

**Comment by gthb96** [comment]
I can confirm the same, noticed it by feeling a warm breeze coming from under my desk in the morning

![image](https://github.com/user-attachments/assets/8a758890-a3d4-4537-a41e-00684d2a4170)
![image](https://github.com/user-attachments/assets/2fee05ec-d499-4faa-a3a5-ce0915182299)

**Comment by gthb96** [comment]
Has anyone tested if newer beta versions still have this problem?

**Comment by Play13thStreet** [comment]
![Image](https://github.com/user-attachments/assets/ff5bcc85-835c-45a5-9522-1067ce8b29d8)

See this, please fix this, for my PC, it can handle it. Ryzen 7 5800X, 32GB RAM 3200, Radeon 7900XT, Watercooled CPU 74°C,
My Laptop, Ryzen 7 4800H, 16GB RAM, Nvidia RTX2060... is very laggy....

**Comment by Felix14-v2** [comment]
Looks like a duplicate of https://github.com/SoftFever/OrcaSlicer/issues/3594 which is fixed in newer versions. Please, make sure that your software is up to date.

**Comment by Orel-A** [comment]
Yes, this has been fixed here, as you can see here: https://github.com/SoftFever/OrcaSlicer/pull/6223
